Portfolio/index.css

```diff
@@ -1,174 +1,126 @@
-console.log('Hello World');

```

Comment:
There is no need for a wildcard selector (*) since it applies styles to all elements. It is better to avoid using wildcard selectors as it can lead to unintended consequences.